### ISSUE TITLE
Add subplebbit.rules edit

### ISSUE THREAD

**Issue by plebeius-eth** [plebbit/plebbit-cli#9]
Currently there's no way to add rules to a subplebbit, and imo this is an essential feature to have as soon as possible.
I think it would be fine if a `--rules` flag required to type the array of strings, ie:
`plebbit subplebbit edit ADDRESS --rules ['rule number one', 'rule number two']` 
I think that's good UX, it's easy to write, and it's especially much faster than being prompted for each rule one at a time.

**Comment by estebanabaroa** [comment]
usually arrays are

```
command --option something --option something --option something
```

which creates an array `['something', 'something', 'something']`

**Comment by Rinse12** [comment]
What about `plebbit subplebbit rules <sub-address> "Rule 1" "Rule 2" "Rule 3"`?

That way it would be consistent with `plebbit subplebbit role set <sub-address> <author-address>`

**Comment by Rinse12** [comment]
What did we agree on at the end? Was it `plebbit subplebbit edit <subplebbit-address> --rules.0 "Rule 0" --rules.2 "Rule 2"`?

**Comment by Rinse12** [comment]
Should be implemented with 0.9 release
